charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -31,7 +31,7 @@ sidecars:
 
 snapshotController:
   repository: k8s.gcr.io/sig-storage/snapshot-controller
-  tag: "v3.0.3"
+  tag: "v4.0.0"
```

Comment:
Same for this one :( https://kubernetes-csi.github.io/docs/snapshot-controller.html